src/agora/agora.cpp

```diff
@@ -1195,13 +1196,13 @@ void Agora::save_tx_data_to_file(UNUSED int frame_id)
     FILE* fp = fopen(filename.c_str(), "wb");
 
     for (size_t i = 0; i < cfg->dl_data_symbol_num_perframe; i++) {
-        int total_data_symbol_id
-            = (frame_id % TASK_BUFFER_FRAME_NUM) * cfg->data_symbol_num_perframe
-            + i + cfg->dl_data_symbol_start;
+        size_t total_data_symbol_id = (frame_id % TASK_BUFFER_FRAME_NUM)
+                * cfg->dl_data_symbol_num_perframe
+            + i;
```

Comment:
Can you use `cfg->get_total_data_symbol_idx_dl(frame_id, i)` here?

Reply:
done!